cmake/openenclave-config.cmake.in

```diff
@@ -73,14 +73,18 @@ elseif (WIN32)
 endif ()
 
 # Include the automatically exported targets.
-include("${CMAKE_CURRENT_LIST_DIR}/openenclave-targets.cmake")
-if (WIN32)
-  include("${CMAKE_CURRENT_LIST_DIR}/add_dcap_client_target.cmake")
-  include("${CMAKE_CURRENT_LIST_DIR}/copy_oedebugrt_target.cmake")
-  include("${CMAKE_CURRENT_LIST_DIR}/maybe_build_using_clangw.cmake")
+if (NOT EXISTS "${CMAKE_CURRENT_LIST_DIR}/openenclave-targets.cmake")
```

Comment:
This check assumes that we can't build both ALL and OEHOSTVERIFY. I would suggest as follows.
- On the root CMakeLists.txt, check if we are building `OEHOSTVERIFY` or `ALL` (e.g., assuming we use `cmake -DCOMPONENT=OEHOSTVERIFY`), and set a cmake variable (e.g., `set(OE_HOST_VERIFY 1)`).
Then like the use of `OE_SGX`, we can use that variable to this check (and the one below).

Reply:
This sounds good. Let me try.